src/Tribe/Load_Assets.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Class used to hook into the assets being loaded
+ *
+ * @since TBD
+ */
+class Tribe__Load_Assets {
```

Comment:
i know @bordoni recommended this class name, but i personally find `load_assets` a bit confusing, since _all_ of our `_asset` classes technically "load assets"

to me something more specific like `modify_assets` or `filter_assets` or even just `assets_loaded` would make more sense, to help clarify that this class and its methods — thoughts?

Reply:
🤔 I think that's a great point. We should probably ask how we see this class developing in the future: if it's only going to be concerned with underscore compatibility, perhaps it ought to follow the same model and naming as our integration classes (in TEC).

Reply:
...if we legit see this as a place to handle all kinds of script conflicts, on the other hand, maybe the name should reflect that?

Reply:
It's mostly a place where we will load all the Assets for Common maybe. Any other suggestions?

Reply:
How about: `Assets_Pipeline`  or `Assets_Sniffer` –– thoughts? Thanks

Reply:
I like Pipeline.